src/types.ts

```diff
@@ -1,4 +1,5 @@
-import {Request} from '@loopback/rest';
+import {Request} from '@loopback/express';
+import PostgresAdapter from 'casbin-pg-adapter';
```

Comment:
why  import like this ? Dont we need {} ?

Reply:
PostgresAdapter is a default export from the casbin-pg-adapter library. That is why such a syntax.